packages/bazel-bot/docker-image/generate-googleapis-gen.sh

```diff
@@ -137,6 +137,11 @@ for (( idx=${#ungenerated_shas[@]}-1 ; idx>=0 ; idx-- )) ; do
     let failed_percent="100 * ${#failed_targets[@]} / $target_count"
     set -e
     echo "$failed_percent% of targets failed to build."
+
+    if [ "${failed_percent}" -gt 10 ]; then
+        echo "More than 10% targets failed. Exiting."
```

Comment:
Should we be more strict with bazelbot failures? If even one API fails to build, and there are user facing changes for said API, we lose the conventional commit message(s) corresponding to those user facing changes. If we don't have the conventional commit message, a release may be missed which is a significant issue.

Reply:
Yes, I'm checking Bazelbot logs to find good value.

Reply:
I checked the logs since November 5th and all are zero. Setting threshold with more than 0 would work.

Reply:
Looks good